apps/product/src/main/java/com/ecommerce/product/dto/ProductDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.ecommerce.product.dto;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.*;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class ProductDto {
+    private Integer id;
+    @NotEmpty
+    private String productName;
+    @NotEmpty
+    private String description;
+    @DecimalMin(value = "1.0")
+    @DecimalMax(value = "1000000.0")
+    private Float price;
+    @NotEmpty
+    private String imageUrl;
+    @Digits(integer = 10, fraction = 0)
+    private Integer quantity;
+    @NotEmpty
+    private String manufacturer;
+    @Pattern(regexp = "^[0-9]{2}-[0-9]{2}-[0-9]{4}$")
```

Comment:
You should leverage the `ZonedDateTime` type from java, and parse the date time string from the service.